pastebin_connector.py

```diff
@@ -63,39 +60,18 @@ def _convert_to_client_tz(self, t_str):
         return t.astimezone(pytz.timezone(client_tz.zone)).strftime("%Y-%m-%d %H:%M:%S %Z")
 
     def _add_file_to_vault(self, action_result, file_name, container_id, file_data):
-        success = True
-
-        if hasattr(Vault, 'get_vault_tmp_dir'):
-            temp_dir = Vault.get_vault_tmp_dir()
-        else:
-            temp_dir = "opt/phantom/vault/tmp"
-
-        temp_dir = temp_dir + '/{}'.format(uuid.uuid4())
-        os.makedirs(temp_dir)
-        fname = os.path.join(temp_dir, file_name)
-
         try:
-            with open(fname, "w") as outfile:
-                outfile.write(smart_str(file_data))
-        except IOError:
-            success = False
-
-        if success:
-            # Save attachment file to the vault
-            try:
-                success, message, attachment_vault_id = ph_rules.vault_add(
-                    container=container_id,
-                    file_location=fname,
-                    file_name=None
-                )
-                if not success:
-                    return action_result.set_status(phantom.APP_ERROR, message)
-                action_result.add_data({"vault_id": attachment_vault_id})
+            resp = Vault.create_attachment(smart_str(file_data), container_id, file_name=file_name)
+        except Exception as ex:
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                f"Unable to add file to the vault for attachment name: "
+                f"{file_name}. Error: "
+                f"{self._get_error_message_from_exception(ex)}"

```

Comment:
```suggestion
                f"Unable to add file to the vault for attachment name: {file_name}. "
                f"Error: {self._get_error_message_from_exception(ex)}"
```

Reply:
Unfortunately can't apply. The lines will exceed the 79 character limit.